scripts/set_aws_secrets.py

```diff
@@ -0,0 +1,277 @@
+    FhirDispensing_proxygen_public_key = read_local_secret(f"{environment}_FhirDispensing_proxygen_public_key")
+    ptl_prescription_signing_public_key = read_local_secret(f"{environment}_ptl_prescription_signing_public_key")
+    ptl_prescription_signing_private_key = read_local_secret(f"{environment}_ptl_prescription_signing_private_key")
+    PSUNotify_callback_api_key = os.environ.get(f"{environment}_PSUNotify_callback_api_key")
+    PSUNotify_callback_app_name = os.environ.get(f"{environment}_PSUNotify_callback_app_name")
+    PSUNotify_kid_secret = os.environ.get(f"{environment}_PSUNotify_kid_secret")
+    PSUNotify_private_key_secret = read_local_secret(f"{environment}_PSUNotify_private_key_secret")
+
+    secret_values: Secrets = {
+        "slack_webhook_url": slack_webhook_url,
+        "splunk_hec_token": splunk_hec_token,
+        "jira_token": jira_token,
+        "confluence_token": confluence_token,
+        "spine_asid": spine_asid,
+        "spine_ca_chain": spine_ca_chain,
+        "spine_public_certificate": spine_public_certificate,
+        "spine_private_key": spine_private_key,
+        "spine_party_key": spine_party_key,
+        "eps_signing_cert_chain": eps_signing_cert_chain,
+        'service_search_api_key': service_search_api_key,
+        "PSU_proxygen_private_key": PSU_proxygen_private_key,
+        "PSU_proxygen_public_key": PSU_proxygen_public_Key,
+        "CPSU_proxygen_private_key": CPSU_proxygen_private_key,
+        'CPSU_proxygen_public_key': CPSU_proxygen_public_key,
+        "ClinicalTracker_proxygen_private_key": ClinicalTracker_proxygen_private_key,
+        "ClinicalTracker_proxygen_public_key": ClinicalTracker_proxygen_public_key,
+        "FhirDispensing_proxygen_private_key": FhirDispensing_proxygen_private_key,
+        "FhirDispensing_proxygen_public_key": FhirDispensing_proxygen_public_key,
+        "ptl_prescription_signing_public_key": ptl_prescription_signing_public_key,
+        "ptl_prescription_signing_private_key": ptl_prescription_signing_private_key,
+        "PSUNotify_callback_api_key": PSUNotify_callback_api_key,
+        "PSUNotify_callback_app_name": PSUNotify_callback_app_name,
+        "PSUNotify_kid_secret": PSUNotify_kid_secret,
+        'PSUNotify_private_key_secret': PSUNotify_private_key_secret
+    }
+    return secret_values
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--environment",
+        required=True,
+        help="Please provide an environment to set secrets in",
+        choices=["dev", "ref", "qa", "int", "prod", "recovery"]
+    )
+
+    arguments = parser.parse_args()
+    # get all the exports for each environment
+    all_exports = get_all_exports(f"prescription-{arguments.environment}")
+    secret_arns = get_secret_arns(all_exports=all_exports)
+    secret_values = get_local_secret_values(all_exports=all_exports)
```

Comment:
I think this function call is wrong - `get_local_secret_values` takes the environment as a string, but you're passing in a list